dockerfile-test

```diff
@@ -0,0 +1,4 @@
+FROM amisevsk/che-plugin-registry:offline-mode
```

Comment:
do we need this file committed? Seems like it's less a test script and more a thing-you-used-for-manual-verification :)

Reply:
It suuure is :)